.tools/ailly/content/start.sh

```diff
@@ -0,0 +1,246 @@
+#!/bin/bash
+
+# Prereq: Install and run Ailly
+# npm install -g @ailly/cli
+
+# Choose your engine, probably bedrock
+export AILLY_ENGINE=bedrock
+
+# Define languages and create directory structures
+languages=("python" "java" "javascript")
```

Comment:
This should be in a separate config file or be passed in as arguments, so we don't have to edit a source controlled file to do different languages.